src/CosmosDBExperiences.ts

```diff
@@ -11,12 +11,14 @@ export enum API {
     MongoDB = 'MongoDB',
```

Comment:
This is pretty minor but I thought we decided on updating this to 'Add firewall rule for *your* IP "x.x.x.x"' just in case the IP we displayed didn't mean much to the user
<img width="509" alt="Screen Shot 2020-05-04 at 11 44 40 AM" src="https://user-images.githubusercontent.com/22795803/81002943-ee73d680-8dfe-11ea-9470-a86059b06458.png">

Reply:
Fixed in different PR